content/topics/kubernetes.md

```diff
@@ -6,265 +6,141 @@ url: /kubernetes
+        - Seamlessly manage cloud resources with the Pulumi Kubernetes Operator.
+        - And more...
+
+    cta: REQUEST MORE INFORMATION
+
+superpowers:
+    - title: Run On Any Cloud
+      cta: Learn more
+      ctaUrl: "/docs/get-started"
+      iconType: cloud
+      description: |
+        With support for all public clouds and dozens of popular infrastructure service
+        providers including private and hybrid clouds, Pulumi gives you the flexibility
+        to run your clusters and workloads where you want to.
+
+    - title: Reduce Provisioning Time
+      cta: Learn more
+      ctaUrl: "/docs/get-started"
+      iconType: provisioning
+      description: |
+        With Pulumi you are able to take advantage of the features of programming
+        languages, helping you reduce boilerplate code and ultimately ship applications
+        and infrastructure faster with greater consistency.
+
+    - title: Automate Delivery
+      cta: Learn more
+      ctaUrl: "/docs/get-started"
+      iconType: delivery
+      description: |
+        You can integrate Pulumi directly with your favorite CI/CD and SCM systems to
+        continuously deliver apps and infrastructure. Improve the velocity and visibility
+        into your deployments from simple to complex global environments.
+
+    - title: Smart Architecture
+      cta: Learn more
+      ctaUrl: "/docs/get-started"
+      iconType: architecture
+      description: |
+        YAML and templated DSLs force you to write the same boilerplate code over and over. Using
+        Pulumi allows you to codify those patterns and best practices so you can stop reinventing
+        the wheel and start inventing the platforms of the future.
+
+    - title: Be Proactive, Not Reactive
+      cta: Learn more
+      ctaUrl: "/docs/get-started"
+      iconType: policy
+      description: |
+        When you enable Pulumi's Policy as Code feature, you instantly gain the power to prevent
+        mistakes from being deployed. Enforce security, compliance, cost controls, and best
+        practices using policies defined in real languages.
+
+    - title: Reduce Deployment Anxiety
+      cta: Learn more
+      ctaUrl: "/docs/get-started"
+      iconType: testing
+      description: |
+        Deploying untested code can lead to some unexpected results. Pulumi lets you take advantage
+        of common tools, frameworks, and techniques to unit, integration, and property test your
+        infrastructure. Ensure your infrastructure is correct before and after deployment.
+
```

Comment:
add a banner with CTA:
"Learn more about Kubernetes at the Cloud Engineering Summit - October 7 & 8" [CTA Button  - "Register Today"] [CTA Link "https://cloudengineering.heysummit.com/?utm_campaign=summit&utm_source=pulumi.com&utm_medium=web"]